packages/sui-react-router/src/ReactRouterContext.js

```diff
@@ -1,17 +1,3 @@
-import React from 'react'
-import hoistNonReactStatics from 'hoist-non-react-statics'
+import {createContext} from 'react'
 
-const ReactRouterContext = React.createContext({})
-
-ReactRouterContext.wrapper = (Component, displayName) => {
```

Comment:
I am using it here: https://github.mpi-internal.com/scmspain/frontend-ma--uilib-components/pull/804/files#diff-4b99e5c03caaee681ecb10f20dcbaab4R173

Reply:
Isn't a possibility to use `contextType` (the one that's not deprecated) to simplify it the library and the change you need to do until you the component is moved to a function?

```
AuthenticateComponent.contextType = RRContext
```